src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -217,32 +348,35 @@ class VisualizeListingTableUi extends Component {
     }));
   }
 
-  closeModal = () => {
-    this.setState({ showDeleteModal: false });
-  };
-
   renderConfirmDeleteModal() {
     return (
       <EuiOverlayMask>
         <EuiConfirmModal
-          title={this.props.intl.formatMessage({
-            id: 'kbn.visualize.listing.deleteVisualizations.confirmModalTitle',
-            defaultMessage: 'Delete selected visualizations?',
-          })}
-          onCancel={this.closeModal}
+          title={
+            <FormattedMessage
+              id="kbn.visualization.listing.deleteSelectedVisualizationsConfirmModal.title"
+              defaultMessage="Delete selected visualizations?"
```

Comment:
We should think about adding the number of selected items to this modal title.

Reply:
done